dask/array/core.py

```diff
@@ -16,6 +16,7 @@
 from threading import Lock
 import uuid
 import warnings
+from .utils import IS_NEP18_ACTIVE
```

Comment:
Moving this import inside the `concatenate3` function should fix the `ImportError`s [currently being raised](https://travis-ci.org/dask/dask/jobs/543666503#L1684)